BUILDING.md

```diff
@@ -36,6 +36,8 @@ $ make
 
 Note that the above requires that `python` resolve to Python 2.6 or 2.7 and not a newer version.
 
+*note: Running make in parallel, ```$ make -j8``` where ```-j8``` specifies that 8 recipes should run concurrently, can significantly reduce build time. See the [GNU Make Documentation](https://www.gnu.org/software/make/manual/html_node/Parallel.html) for more information.*
+
```

Comment:
I'm guessing that most people won't know what a recipe is so maybe make that "compile job" or something like that?

Style nit: can you wrap lines at 80 columns?  You also don't need to use triple backticks, single ones suffice.  I wouldn't necessarily use italics either but that's more of a personal preference.